src/index.html

```diff
@@ -6,13 +7,69 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
```

Comment:
The `preconnect` to fonts.gstatic.com is unnecessary if you're using fonts.googleapis.com, as the latter should implicitly handle the connection to the font service.